future/name-resolution/recursive-dns/README.md

```diff
@@ -625,7 +625,135 @@ NOTE: We hacked this together a little... We used glue records in the zonefile w
+Interestingly, it doesn't do the same for `com`. This is likely because we didn't clear the cache on our whole internet. If we were to restart each DNS server on our internet, we'd likely see requests for `com` to both root DNS servers.
+
+The next 3 lines are the responses back from the root DNS servers. The actual response bodies aren't parsed here, but given what we saw in the exercises above, we know that what should be seeing DNS responses for where the `com` and `org` TLD servers are. These responses will include the IP addresses for those servers.
+
+
+```bash
+21:11:07.567766 IP 1.2.0.100.63263 > 8.2.0.100.53: 3546% [1au] A? awesomecat.com. (43)
+```
+
+`1.2.0.100` (`resolver-c`) sends a request to `8.2.0.100` (`tlddns-g`) for the `A` records for `awesomecat.com`.
+
+```bash
+21:11:07.568250 IP 1.2.0.100.64823 > 8.2.0.100.53: 5986% [1au] A? google.com. (39)
+```
+
+When `resolver-c` learned about the `com` TLD, it received an `NS` record that pointed it to `tlddns-g.google.com`. Our resolver is a diligent machine, so it wants to go ahead and learn everything it can about the domains it's given. So, it's gonna go ahead and fire off a request to gather the `A` records for `google.com` while it's at it.
+
+There are more requests that are not directly tied to resolving `www.awesomecat.com`. Instead of including those in the line by line analysis, they're gonna be in a block at the end. That way they can still be documented to see the work the resolver is doing without detracting from this analysis.
+
+```bash
+21:11:07.568496 IP 8.2.0.100.53 > 1.2.0.100.63263: 3546- 0/1/2 (93)
+21:11:07.568758 IP 8.2.0.100.53 > 1.2.0.100.64823: 5986- 0/1/2 (89)
+```
+
+`8.2.0.100` (`tlddns-g`) responds back to the queries `1.2.0.100` (`resolver-c`) just made. In each of these lines, you'll see a request ID: `3546` and `5986`. If you check the previous requests coming from `resolver-c`, you can find the corresponding request ID. Again, from our exercises above, we can surmise that response `3546` tells our intrepid resolver that the authoritative DNS server for `awesomecat.com` is `authoritative-a.aws.com` AND it provides the IP address for that server.
+
+```bash
+21:11:07.571062 IP 1.2.0.100.17366 > 4.1.0.100.53: 30044% [1au] A? www.awesomecat.com. (47)
+```
+
+`1.2.0.100` (`resolver-c`) sends a request to `4.1.0.100` (`authoritative-a`) for the `A` records for `www.awesomecat.com`. Remember, `resolver-c` learned about the IP address for the authoritative server for `awesomecat.com` in request `3546` above.
+
+```bash
+21:11:07.571741 IP 4.1.0.100.53 > 1.2.0.100.17366: 30044*- 1/0/1 A 4.2.0.11 (63)
+```
+
+`4.1.0.100` (`authoritative-a`) responds to `1.2.0.100` (`resolver-c`) with the `A` record (IPv4 address) for `www.awesomecat.com`.
+
+```bash
+21:11:07.572238 IP 1.2.0.100.53 > 1.1.0.200.48600: 48085 1/0/1 A 4.2.0.11 (63)
+```
+
+HERE IT IS! `1.2.0.100` (`resolver-c`) responds to `1.2.0.100` (`client-c1`) with the `A` record for `www.awesomecat.com`!!!
```

Comment:
It seems like line 705 is the *HERE IT IS!* moment where the resolver finally gets the IP address for the domain it's been hunting. Line 711 is the resolver responding back to the client. I think you say all this in the text, but I would like to see a bit more emphasis on line 705.